packages/schematics/angular/application/other-files/app.module.ts

```diff
@@ -15,7 +15,7 @@ import { environment } from '../environments/environment';<% } %>
   imports: [
     BrowserModule<% if (routing) { %>,
     AppRoutingModule<% } %><% if (serviceWorker) { %>,
-    environment.production ? ServiceWorkerModule.register('/ngsw-worker.js') : []<% } %>
+    ServiceWorkerModule.register('/ngsw-worker.js', {enabled: environment.production})<% } %>
```

Comment:
Only comment: spaces between `{}` and its content.

Reply:
Done.